xml/System.Reflection/EventInfo.xml

```diff
@@ -160,22 +160,21 @@
 ## Remarks  
  This method attempts to add a delegate to synchronize the event on the target object.  
   
- Each time the event is raised by the target parameter, the method or methods encapsulated by the handler will be invoked.  
+ Each time the event is raised by the target parameter, the method or methods encapsulated by the handler are invoked.  
   
-   
+You might use the `AddEventHander` method when you load a type after the program is already compiled, when it is not possible to use the C# += syntax or the Visual Basic `WithEvents` and `Handles` keywords to hook up the event.   
```

Comment:
This technically doesn't apply anymore to the modified sample, since `typeof()` requires that the type is known during compilation.

Does that matter?